src/content/pages/faq.mdx

```diff
@@ -17,6 +17,7 @@ subtitle: Frequently Asked Questions about EuroPython
   <li>**5 February 2025:** Proposal Review process starts</li>
   <li>**6 February 2025:** [Community Voting](/programme/voting/) starts</li>
   <li>**17 February 2025:** Community Voting and Review ends</li>
+  <li>**undefined 2025:** [Sponsorship](/sponsor) Sign-up Starts </li>
```

Comment:
Might be better to just mention the month?

Reply:
all undefined are replaced with real ~~dogs~~ dates now <3